File: Objective-C/Reference App/Support/MainMenuViewController.m
//
//  MainMenuViewController.m
//  Reference App
//
//  Created by David Deller on 5/3/18.
//  Copyright © 2018 TallyGo. All rights reserved.
//

#import "MainMenuViewController.h"

@interface MainMenuViewController ()

@end

@implementation MainMenuViewController

- (void)viewDidAppear:(BOOL)animated {
    [super viewDidAppear:animated];
    
    // Clean up by resetting any settings from examples back to defaults
    TGDrivingSimulator.sharedDrivingSimulator.enabled = NO;
}

@end

File: Swift 4/Reference App/Support/MainMenuViewController.swift
//
//  MainMenuViewController.swift
//  Reference App
//
//  Created by David Deller on 4/30/18.
//  Copyright © 2018 TallyGo. All rights reserved.
//

import UIKit
import TallyGoKit

class MainMenuViewController: UITableViewController {
    
    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
        
        // Clean up by resetting any settings from examples back to defaults
        TallyGo.simulatedCoordinate = kCLLocationCoordinate2DInvalid
    }
    
}
